podman/domain/containers.py

```diff
@@ -214,6 +214,16 @@ def get_archive(
         stat = api.decode_header(stat)
         return response.iter_content(chunk_size=chunk_size), stat
 
+    def inspect(self) -> None:
```

Comment:
"-> None" is probably not correct. Can you fix that to reflect what the json object is

Reply:
I'm looking at this, and obviously you're correct - `None` is wrong. However, it returns a dict of a lot of types - would it be best to just have it be:
```py
def inspect(self) -> Dict:
```
Or would a different approach be preferred?

Reply:
Yea dict would be fine.

Reply:
Added as requested